addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -483,3 +483,17 @@ img.o_we_custom_image.mx-auto {
     flex: 1 1 auto;
     outline: none;
 }
+
+// Color picker dialog for JW Editor
+.o_colorpicker_container{
+    background-color: #000;
+    position: absolute;
+    top: 45px;
+    left: 45px;
+    z-index: 100;
+    width: 230px;
+    .btn, .o_color_picker_inputs{
+        color: white;
+        background-color: #444;
+    }
+}
```

Comment:
Seem a bit too much ad hoc for me.
- The toolbar could be anywhere.
- The z-index could break anytime.
- The size could be different if the colorpicker size change.
- Shouldn't we use the colors of Odoo?

Reply:
Yes this are all valid remarks.
I mainly added this css to "make it work" before we make a big pass on the look and feel of the JW integration.
But I can change it if you want.